src/EventStore.Core/Services/Transport/Grpc/Streams.Read.cs

```diff
@@ -76,6 +76,8 @@ internal partial class Streams<TStreamId> {
 					}
 				} catch (ReadResponseException ex) {
 					ConvertReadResponseException(ex);
+				} catch (Exception) {
```

Comment:
We should only trap the specific exception that's flooding the logs

Reply:
Done